docs/src/pages/quasar-cli/routing.md

```diff
@@ -28,7 +28,7 @@ export default function ({ store /*, ssrContext */ }) {
 ```
 
 
-Since you cannot import Vuex from other scripts you can make store accessable to your routes array, by creating an imported routes function that takes the store as an argument. This allows you to do per route basis checking against the Vuex state. For example to see if the user has a subscription. 
+You can make the store accessable to your routes array by creating an imported routes function that takes the store as an argument. This allows you to do per route basis checking against the Vuex state. For example to see if the user has a subscription. 
```

Comment:
spelling | accessable => accessible
Should `imported` in this case be mentioned as `exported`?
Remove the word `basis` => `per route checking...` is sufficient
`For example to see if the user has a subscription.` I don't think this is needed at all. If you disagree, then it needs to be written better.

Reply:
I actually agree here. I didn't realize that I was going around my elbow to accomplish what I needed to get done.